dbeaver/dbeaver
# dbeaver/pro#1862 Redirect awt browse requests to swt browser

Closes dbeaver/pro#1862
I hope this will work...
![image](https://github.com/dbeaver/dbeaver/assets/20579256/b7f6e5a0-0791-4628-87aa-789f70d969f7)

Edge cases to test:
1. Try running with system java
2. Try running on Linux/MacOS distributions
3. Try different OAuth, excluding AWS(due to it calling auth via launching another binary)
4. Test that everything works in TE
5. Check that the checkbox is present for all systems and should be disabled if Internet Explorer is selected. No redirection should happen if IE is the default browser
